httpbakery/client.go

```diff
@@ -222,6 +222,9 @@ func (c *Client) DoWithBody(req *http.Request, body io.ReadSeeker) (*http.Respon
 }
 
 func (c *Client) doWithBody(req *http.Request, body io.ReadSeeker) (*http.Response, error) {
+	if req.Body != nil {
+		return nil, errgo.New("body unexpectedly supplied in Request")
```

Comment:
more explicit. req.body unexpectedly supplied in request

(also note lower case r in request)